etc/profile-m-z/onionshare-gui.profile

```diff
@@ -34,8 +35,13 @@ novideo
 protocol unix,inet,inet6
 seccomp
 shell none
+tracelog
 
 private-dev
 private-tmp
 
+dbus-user filter
+dbus-user.talk org.freedesktop.Notifications
```

Comment:
I added it because 
```
$ flatpak remote-info --system flathub -m org.onionshare.OnionShare
…
[Session Bus Policy]
com.canonical.AppMenu.Registrar=talk
org.freedesktop.Notifications=talk
org.freedesktop.secrets=talk
```

Reply:
In that case better add that too. Thanks for the flathub reminder!